cinch/roles/jenkins_common/defaults/main.yml

```diff
@@ -1,10 +1,8 @@
-jswarm_version: 2.1
-jenkins_slave_username: ''
-jenkins_slave_password: ''
 # The home directory for the user who will be executing the swarm process
 jenkins_user_home: /var/lib/jenkins
 # The name of the user who will be executing the swarm process
 jenkins_user: jenkins
+jenkins_user_password: changeme
```

Comment:
Before this change, the jenkins user was being created both in jenkins_common and in jenkins_slave, using the same username (jenkins), with the difference that in jenkins_common you specified more details, but no password and in jenkins_slave you specified just username and password. Since it's the same user, I decided to merge this to create the user in one task here. Maybe I misunderstood and the password should only be set on slave, in that case I'd suggest reverting this and renaming the task from "create jenkins user" to "change password for jenkins user"